dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -174,6 +175,14 @@
 				clocks = <&rcc STM32_CLOCK(IOP, 3U)>;
 			};
 
+			gpioe: gpio@50001000 {
+				compatible = "st,stm32-gpio";
+				gpio-controller;
+				#gpio-cells = <2>;
+				reg = <0x50001000 0x400>;
+				clocks = <&rcc STM32_CLOCK(IOP, 4U)>;
+			};
```

Comment:
GPIOE is only available for g0b0/1 and g0c1 variants.

Reply:
Right. I checked a couple of datasheets and had the wrong impression that it depends on the package size only (which is currently not considered in the devicetree file naming schema, as far as I know).

Hope it's ok to define WKUP5 in the base file even though it might not be available in small packages.